remix-solidity/src/compiler/compiler-worker.ts

```diff
@@ -1,26 +1,23 @@
 'use strict'
 
-const solc = require('solc/wrapper')
-
-let compileJSON = function () { return '' }
-const missingInputs = []
-
-module.exports = function (self) {
-  self.addEventListener('message', (e) => {
-    const data = e.data
+import solc from 'solc/wrapper'
+import { CompilerInput, MessageToWorker } from './types'
+var compileJSON: ((input: CompilerInput) => string) | null = (input) => { return '' }
+var missingInputs: string[] = []
+
+export default (self) => {
+  self.addEventListener('message', function (e) {
```

Comment:
The only reason to use a normal function here is if you want to access the element that triggered the `message` event. But I don't see it here, so I would recommend using arrow function (espacially if you consider the synthax above).

Reply:
Yes, thanks!